src/game.h

```diff
@@ -411,7 +411,7 @@ class game
 
         void open_gate( const int examx, const int examy, const ter_id handle_type );
 
-        bionic_id random_good_bionic() const; // returns a non-faulty, valid bionic
+        std::string random_good_bionic() const; // returns a non-faulty, valid bionic
```

Comment:
Yeah, those low level ones are the ones I was thinking of. When making the changes I took the path of least resistance to leave things as they were. If others think that those functions ought to be moved into `bionics.cpp` I see no big problem either.